src/troubleshooting.html

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
why are we adding this back?

Reply:
Uh, that was unintentional, probably a (back) merge artifact.